cognite_toolkit/_cdf_tk/commands/_changes.py

```diff
@@ -52,40 +50,6 @@ def instructions(self, files: set[Path]) -> str:
         return ""
 
 
-class SetKindOnFile(AutomaticChange):
-    """Adds the kind to the filename of all resource files.
-
-Before `your_file.yaml`:
-After `your_file.FileMetadata.yaml`:
-    """
-
-    deprecated_from = Version("0.4.0")
-    has_file_changes = True
-
-    def do(self) -> set[Path]:
```

Comment:
This auto change uses the old logic we should not continue to use. Suggest we just delete it and say that we do not support upgrade before v0.4.